src/OperationalInsights/OperationalInsights.Autorest/examples/Get-AzOperationalInsightsDeletedWorkspace.md

```diff
@@ -0,0 +1,72 @@
+## EXAMPLES
+
```

Comment:
```suggestion
```

Reply:
Please refer to the format from https://raw.githubusercontent.com/Azure/azure-powershell/main/src/ContainerInstance/examples/Get-AzContainerGroup.md